Week-2/Exercise-2.1/index.html

```diff
@@ -0,0 +1,93 @@
+            <h2>What is Lorem Ipsum?</h2>
+            <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard
+            dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen
+            book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially
+            unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more
+            recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
+        </article>
+        
+        <article class="article">
+            <h2>Why do we use it?</h2>
+            <p>It is a long established fact that a reader will be distracted by the readable content of a page when looking at its
+            layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using
+            'Content here, content here', making it look like readable English. Many desktop publishing packages and web page
+            editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites
+            still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose
+            (injected humour and the like).</p>
+        </article>
+        <div class="img-wrapper">
+            <img width="300px" height="300px" src="https://uploads.codesandbox.io/uploads/user/a8233562-b36b-483d-9de9-5a1ac148fb0c/2Eq--cattyboard.jpg" alt="cattyboard">
+        </div>
+        <article class="article">
+            <h2>Where does it come from?</h2>
+            <p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature
+            from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia,
+            looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of
+            the word in classical literature, discovered the undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33
+            of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a treatise
+            on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit
+            amet..", comes from a line in section 1.10.32.</p>
+        </article>
+
+        <form action="" method="post" class="form">
+            <label for="title">Title</label><br>
+            <input type="radio" id="mr" name="title" value="Mr">
+            <label for="mr">Mr</label>
+            <input type="radio" id="miss" name="title" value="Miss">
```

Comment:
@AakashPandya Form elements are not aligned properly. Looking very odd, can you provide some spacing and arrange it accordingly

Reply:
Thanks @RajeshSivanesan , formatted html, css both files by using prettier code formatter extension. 
I will take care about this in future.